README.md

```diff
@@ -45,189 +9,88 @@ After you're finished please remove all the comments and instructions!
   - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
   - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 HTML/CSS & JavaScript capstone<a name="about-project"></a>
 
-**[your_project__name]** is a...
+**[HTML/CSS & JavaScript capstone]** is a the final project of Microverse Program: Module .  I am going to build an online website for a conference. Applying all my acknowledgment about HTML/CSS & JavaScript.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://html.com/">Html</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">Css</a></li>
+    <li><a href="https://www.javascript.com/">Figma</a></li>
   </ul>
 </details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Create web pages from scratch using Css, html & JavaScript]**
+- **[Use JavaScript to manipulate DOM elements.]**
+- **[Use GitHub Pages to deploy web pages.]**
+- **[Apply JavaScript best practices and language style guides in code.]**
+- **[Communicate technical concepts to other technical people.]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Enjoy](https://nkaleth.github.io/capstone_project1/) 😃 
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better to use a more descriptive name for your Link instead of just `Enjoy`. Something like `Link to Live demo` will be easier to understand. or what do you think?? 🌸